### DISCUSSION TITLE
Github Copilot doesn't work in Visual Studio Community 2022 on main development desktop

### DISCUSSION THREAD

**Discussion by Dessyreqt** [community/community#61012] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

When I start up Visual Studio 2022 on my desktop PC, Copilot doesn't start automatically:
![image](https://github.com/community/community/assets/12574786/d71e6ca6-1fd2-4418-93f1-63d6e1cd8434)

I can get it to sign in by unchecking "Enable Globally" and then rechecking it:
![image](https://github.com/community/community/assets/12574786/3bd9e164-9941-45c5-83ec-c28757c22414)

However, it doesn't provide any code suggestions, and hovering over the icon says that it hasn't been started:
![image](https://github.com/community/community/assets/12574786/5d1070ca-899e-4aa0-9303-db533ded5739)

Additionally, if I try to Open Copilot from the icon, I'm greeted with the following popup:
![image](https://github.com/community/community/assets/12574786/1df2cd5e-479b-46af-b2e1-8b1daf7a81de)

Nothing else is logged to the output window, and I don't know of any other logs that I can check to find the source of this error. I have Copilot working in VS Code on the same machine, and in both VS2022 Community and VS Code on my laptop, but I would like to get it working on my main development IDE on my main development computer. Any help would be appreciated.

**Comment by Adn990** [comment]
mine is the same, i will cancel subscription if not fixed. more annoying.

**Reply by ayim** [reply]
@Adn990 @dessyreqt we'd love to fix this - if you use the report a problem feature in VS 
![image](https://github.com/community/community/assets/10951869/53d0ee2f-4d4a-434b-a896-4f121b036daa)

and make sure attach system logs is checked
![image](https://github.com/community/community/assets/10951869/a888e377-4451-48f7-91a9-844cde6fb2de)

We'll have the logs we need to diagnose, thanks!
